### DISCUSSION TITLE
Me aparecen 2 errores

### DISCUSSION THREAD

**Discussion by Edgarcastrop** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#473] [Q&A]
Disculpen la molestia, pero cada que pongo un Pokémon en el input, me aparecen 2 errores. Me podrían dar sus observaciones

![Capturas](https://user-images.githubusercontent.com/99065409/159189073-096b043f-b445-4992-86e4-1bd33a097f00.JPG)

![Capturas1](https://user-images.githubusercontent.com/99065409/159189057-72019a32-a262-414e-9233-b2d360b034fa.JPG)
r

**Comment by AngelCruzO** [comment]
cambia el valor de ${pokeName}, por ${pokeInput}. No le estas pasando el valor

**Reply by Edgarcastrop** [reply]
Muchas gracias, buen hombre

**Comment by doguedogue** [comment]
En tu URL que haces fetch usa pokeInput

${ pokeInput }

**Reply by Edgarcastrop** [reply]
Muchas gracias, héroe
